doc/faqs.md

```diff
@@ -1,5 +1,14 @@
 # FAQs
 
+## Debug
+
+### Q: Experiment not working after apply chaos
+You can following the steps to debug:
+- First using `kubectl describe` the chaos experiment resource. If see `NextStart`  and `NextRecover`  field in spec, then after `NextStart` the chaos will trigger.
+- Then use `kubectl logs -f -n chaos-testing chaos-controller-manager-xxxxx(replace this will controller-manager's pod name)`  to get controller-manager's log and see whether there are errors in it, If there are some related errors, please file an issue.
+- If not the above two cases, then you can use `kubectl get pods -n yourNamespace(using your namespace) --show-labels` to show the labels and check if it is meet selector.
```

Comment:
If the pods didn't meet selector, `no pod is selected`  error message will be printed at the controller log